CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- Use Heartbeat v2 API
```

Comment:
This is a needs to go under a section header such as `### Changed`, `### Added`, `### Breaking Changes`, `### Fixed` as outlined [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md). Can you verify that the api key provided for v1 works on v2 or do they have different auth mechanisms. I am just asking because I have seen that kind of stuff before and I am not really familiar with `opsgenie`

Reply:
I not able to verify whether the same API key can be used for v1 and v2; I only have access to the v2 APIs.  Attempting to use my current API key for the v1 Heartbeat API results in an HTTP 401 response.

Reply:
@eheydrick can you check on this since I belive you use this?

Reply:
Updated PR to list this change under a "Fixed" section.